sdks/java/maven-archetypes/examples/build.gradle

```diff
@@ -17,10 +17,59 @@
  */
 
 apply from: project(":").file("build_rules.gradle")
-applyJavaNature()
+applyJavaNature(artifactId: "beam-sdks-java-maven-archetypes-examples")
 
 description = "Apache Beam :: SDKs :: Java :: Maven Archetypes :: Examples"
 
+// Extract the version from a library:version dependency.
+def ver = { moduleName ->
```

Comment:
Note that defining really short names like `ver` can lead to naming conflicts since ver becomes `global` within this project.

I might be better to just invoke the one liner `dependencies.create(moduleName).getVersion()` as needed directly.

Reply:
Done, made it inline and deleted the ver() function